packages/terra-date-picker/src/terra-dev-site/doc/guides/DatePickerEvents.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import DatePicker from '../../../DatePicker';
+
+class DatePickerEvents extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      blurTriggerCount: 0,
+      focusTriggerCount: 0,
+      changeValue: '',
+      changeRawValue: '',
+      selectValue: '',
+      clickOutsideTriggerCount: 0,
+    };
+    this.handleBlur = this.handleBlur.bind(this);
+    this.handleFocus = this.handleFocus.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+    this.handleChangeRaw = this.handleChangeRaw.bind(this);
+    this.handleSelect = this.handleSelect.bind(this);
+    this.handleOnClickOutside = this.handleOnClickOutside.bind(this);
+    this.blurCount = 0;
+    this.focusCount = 0;
+    this.clickOutsideCount = 0;
+  }
+
+  handleBlur() {
+    this.blurCount = this.blurCount + 1;
+    this.setState({ blurTriggerCount: this.blurCount });
+  }
+
+  handleFocus() {
+    this.focusCount = this.focusCount + 1;
+    this.setState({ focusTriggerCount: this.focusCount });
+  }
+
+  handleChange(event, value) {
+    this.setState({ changeValue: value });
+  }
+
+  handleChangeRaw(event, vaue) {
+    this.setState({ changeRawValue: vaue });
+  }
+
+  handleSelect(event, value) {
+    this.setState({ selectValue: value });
+  }
+
+  handleOnClickOutside() {
+    this.clickOutsideCount = this.clickOutsideCount + 1;
+    this.setState({ clickOutsideTriggerCount: this.clickOutsideCount });
+  }
+
+  render() {
+    return (
+      <div>
+        <p>
+          onBlur Trigger Count:
```

Comment:
nit: for readability, let's add a space after each p string
```suggestion
          onBlur Trigger Count: 
```

Reply:
Updated: https://github.com/cerner/terra-framework/pull/588/commits/608599424d554dff2a4908132605c72032d51661